src/test/java/gdsc/konkuk/platformcore/controller/attendance/AttendanceControllerTest.java

```diff
@@ -141,7 +142,7 @@ void should_get_events_of_the_month_when_pass_year_month() throws Exception {
     void should_attend_when_pass_event_id_and_member_id() throws Exception {
         // given
         Member memberToAttend = MemberFixture.builder()
-                .email("ex@gmail.com").build().getFixture();
+                .email("ex@gmail.com").role(MemberRole.MEMBER).build().getFixture();
```

Comment:
```suggestion
                .email("ex@gmail.com")
                .role(MemberRole.MEMBER).build().getFixture();
```
이거 포맷팅때문에 한줄로 된걸까요?? 다른곳은 두줄인데 이부분은 한줄이네요!

Reply:
피드백 반영한 후 전체 포매팅하면서 요런 부분 가능한 일치시켜볼게욥